certs/ed25519/client-ed25519-priv.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
All new files need to be added to an .am file.

Reply:
Done.